src/OrchardCore.Modules/OrchardCore.Media.AmazonS3/Manifest.cs

```diff
@@ -0,0 +1,19 @@
+using OrchardCore.Modules.Manifest;
+
+[assembly: Module(
+    Name = "Amazon S3 Media",
+    Author = ManifestConstants.OrchardCoreTeam,
+    Website = ManifestConstants.OrchardCoreWebsite,
+    Version = ManifestConstants.OrchardCoreVersion
+)]
+
+[assembly: Feature(
+    Id = "OrchardCore.Media.AmazonS3.Storage",
```

Comment:
@neglectedvalue Just for info

When a module doesn't define any feature, in fact it is like it defines only one feature whose Id equals to the module Id (the assembly name most of the time equals to the project/package name), here `OrchardCore.Media.AmazonS3`.

When a module define at least one feature, we are used to set the Id of the first feature to the module assembly name, one difference is that for some definitions as Controlers, Migrations, Startups (as in your case) you then don't need to decorate them with the `[Feature("...")]` attribute.

But yes, the existing `OrchardCore.Media.Azure` doesn't follow this "rule", maybe this is the only one ;)

So as you want, this was just for info.

Reply:
> But yes, the existing OrchardCore.Media.Azure doesn't follow this "rule", maybe this is the only one ;)

I may check the other once and fix

Reply:
Thanks for the update

For the module attribute you don't need to set the Id, it is modifiable for people defining a different assembly name in their project file, so here would need to be `Id = "OrchardCore.Media.AmazonS3"` but useless.

But always better to define a feature, even if there is only one, so that it is simpler if you add more features.

**So you can just keep what you did before but just replace for this only one feature** (so the 1st one) this

    Id = "OrchardCore.Media.AmazonS3.Storage",

with

    Id = "OrchardCore.Media.AmazonS3",

Reply:
using OrchardCore.Modules.Manifest;

    [assembly: Module(
        Name = "Amazon S3 Media",
        Author = ManifestConstants.OrchardCoreTeam,
        Website = ManifestConstants.OrchardCoreWebsite,
        Version = ManifestConstants.OrchardCoreVersion
    )]

    [assembly: Feature(
        Id = "OrchardCore.Media.AmazonS3",
        Name = "Amazon S3 Media Storage",
        Description = "Enables support for storing media files in Amazon S3 Bucket.",
        Dependencies = new[]
        {
            "OrchardCore.Media.Cache"
        },
        Category = "Hosting"
    )]

Reply:
Yeap, done, thx for that info.